csharp/ql/src/semmle/code/csharp/ExprOrStmtParent.qll

```diff
@@ -346,7 +346,7 @@ private int getImplementationSize(ValueOrRefType t, File f) {
     result = getImplementationSize1(t, f)
 }
 
-private cached module Cached {
+cached module Internal {
```

Comment:
Non- `private` generic names like `Internal` make imports conflict. Rename?

Reply:
Could rename to `ExprOrStmtParentInternal`, but I don't expect code to directly import this file, except those places in the library where we have already imported it.